src/constructs/autoscaling/asg.test.ts

```diff
@@ -114,19 +114,29 @@ describe("The GuAutoScalingGroup", () => {
   test("does not include the UpdatePolicy property", () => {
     const app = new App();
     const stack = new GuStack(app);
-    new GuAutoScalingGroup(stack, "AutoscalingGroup", defaultProps);
+    new GuAutoScalingGroup(stack, "AutoscalingGroup", { ...defaultProps, overrideId: true });
```

Comment:
I think this test should pass regardless of the overrideId flag so maybe it's not needed here?

Reply:
Unfortunately not, as `Object.keys(json.Resources.AutoscalingGroup)` fails because there is no `AutoscalingGroup` key in `Resources` since it'll be auto-generated.

Reply:
Ah, good point!